lib/query-options.js

```diff
@@ -0,0 +1,151 @@
+const rust = require("../index");
+
+/**
+ * Query options
+ * @typedef {Object} QueryOptions
+ * [TODO: Add support for this field]
+ * @property {Boolean} [autoPage] Determines if the driver must retrieve the following result pages automatically.
+ *
+ * This setting is only considered by the [Client#eachRow()]{@link Client#eachRow} method. For more information,
+ * check the
+ * [paging results documentation]{@link https://docs.datastax.com/en/developer/nodejs-driver/latest/features/paging/}.
+ *
+ * [TODO: Add support for this field]
+ * @property {Boolean} [captureStackTrace] Determines if the stack trace before the query execution should be
+ * maintained.
+ *
+ * Useful for debugging purposes, it should be set to ``false`` under production environment as it adds an
+ * unnecessary overhead to each execution.
+ *
+ * Default: false.
+ * [TODO: Add support for this field]
+ * @property {Number} [consistency] [Consistency level]{@link module:types~consistencies}.
+ *
+ * Defaults to ``localOne`` for Apache Cassandra and DSE deployments.
+ * For DataStax Astra, it defaults to ``localQuorum``.
```

Comment:
:wrench: Is this TODO justified? This PR already makes the statement set the chosen consistency level.
Also, the defaults are no longer true, because Rust driver has different defaults and you override them with no default value on the JS side.
Let's remove mentions of DSE and Astra; Rust driver has one default `Consistency` for all cases.